data/json/items/gun/223.json

```diff
@@ -45,6 +45,31 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "ar_pistol",
+    "copy-from": "rifle_semi",
+    "looks_like": "skorpion_61",
+    "type": "GUN",
+    "name": { "str": "AR pistol" },
+    "description": "A compact, 7.5 inch barrel version of the classic AR-15 design, commercially marketed as a home defense weapon.",
+    "weight": "2267 g",
+    "volume": "1758 ml",
+    "price": 91400,
+    "to_hit": -2,
+    "bashing": 9,
+    "material": [ "steel", "aluminum" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": "223",
+    "skill": "pistol",
+    "range": -6,
+    "ranged_damage": -16,
+    "dispersion": 380,
+    "durability": 7,
```

Comment:
6, or lower. AR's become increasing unfond of being overgassed, which pistol length systems tend to do.